x-pack/legacy/plugins/siem/public/pages/detection_engine/components/signals_histogram_panel/helpers.tsx

```diff
@@ -27,6 +31,8 @@ export const formatSignalsData = (
   }, []);
 };
 
+export const memoFormatSignalsData = memoizeOne(formatSignalsData, deepEqual);
```

Comment:
instead of doing this here, can we do the memoization where we do the fetch? So when other engineers will use the   hooks will be already memoized.